DOCS/man/input.rst

```diff
@@ -2691,8 +2691,9 @@ Property list
     ``video-params/crop-w``, ``video-params/crop-h``
         Video size after cropping.
 
-    ``video-params/aspect``
-        Display aspect ratio as float.
+    ``video-params/aspect`` (RW)
```

Comment:
Shouldn't this be `--vf=format=aspect`? video-params has always be read only observable values.

Reply:
Implement it in `--vf=format` you mean? But that doesn't solve the issue. You would still need some special value that means "don't override the aspect ratio" (probably 0 in this case). e.g. if you want to `multiply number aspect-ratio` you'll get 0 with a `vf=format=aspect` instead of -1 (`video-aspect-override`) assuming you start mpv with all default values.

Reply:
Alternatively, we could bring back `video-aspect` as property (heh).